File: control-assurance-sp/src/types/CLHiringMember.ts
import { IEntity } from "./Entity";

export interface ICLHiringMember extends IEntity {
    CLCaseId?: number;
    UserId?: number;
}

export class CLHiringMember implements ICLHiringMember {
    public ID = 0;
    public Title = null;
    public CLCaseId = null;
    public UserId = null;
}

File: ControlAssuranceAPI/Models/CLHiringMember.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class CLHiringMember
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? CLCaseId { get; set; }

    public int? UserId { get; set; }

    public DateTime? DateAssigned { get; set; }

    public virtual CLCase? CLCase { get; set; }

    public virtual User? User { get; set; }
}
